### PR TITLE
Kraken: move last_rt_data_loaded in kraken

### PR DESCRIPTION
**Pull request by xlqian**
make last_rt_data_loaded more meaningful..

### CODE REVIEW

File: source/kraken/maintenance_worker.cpp [hunk 1]
```diff
@@ -198,6 +197,7 @@ void MaintenanceWorker::handle_rt_in_batch(const std::vector<AmqpClient::Envelop
         LOG4CPLUS_INFO(logger, "rebuilding data raptor");
         data->build_raptor(conf.raptor_cache_size());
         data_manager.set_data(std::move(data));
+        data->last_rt_data_loaded = pt::microsec_clock::universal_time();
         LOG4CPLUS_INFO(logger, "data updated " << envelopes.size() << " disrutpion applied in "
```
Comment (pbougue) [inline source/kraken/maintenance_worker.cpp]: minor: typo `disruption`

**Review by pbougue — APPROVED** [review]

**Comment by pbougue** [comment]
Jenkins, test this please

File: source/kraken/maintenance_worker.cpp [hunk 2]
```diff
@@ -172,8 +172,7 @@ void MaintenanceWorker::handle_rt_in_batch(const std::vector<AmqpClient::Envelop
         for(const auto& entity: feed_message.entity()){
             if (!data) {
                 data = data_manager.get_data_clone();
-                data->last_rt_data_loaded = pt::microsec_clock::universal_time();
-                LOG4CPLUS_INFO(logger, "data copied in " << (data->last_rt_data_loaded - begin));
+                LOG4CPLUS_INFO(logger, "data copied in " << (pt::microsec_clock::universal_time() - begin));
```
Comment (pbougue) [inline source/kraken/maintenance_worker.cpp]: can you create/use an other "begin" (to avoid counting the parsing in the copy) ?

**Comment by buildbotctp** [comment]
Build finished.

**Comment by buildbotctp** [comment]
Build finished.

**Pull request merged**